src/Commands/MachineToken/DeleteCommand.php

```diff
@@ -15,14 +16,27 @@ class DeleteCommand extends TerminusCommand
      * Removes a machine token from the logged-in user's account
      *
      * @name machine-token:delete
+     * @param string $machine_token_id The ID of the machine token to be deleted
+     * @throws \Terminus\Exceptions\TerminusException
      * @aliases mt:delete
      *
-     * @param machine_token_id The ID of the machine token to be deleted
      * @usage terminus machine-token:delete <machine_token_id>
      *   Removes the given machine token from the user's account
      */
     public function delete($machine_token_id) {
+        $user = $this->session()->getUser();
 
+        // Find the token. Will throw an exception if it doesn't exist.
+        $machine_token = $user->machine_tokens->get($machine_token_id);
```

Comment:
Here I'm letting the exception thrown by `get()` fall through and get caught by the runner. This works fine (and saves a few lines of code) but the Robo runner isn't familiar with our translatable exceptions. That means the output when there is an error here is:

[error]  Could not find {model} "{id}" 

We have 3 options:

1) Catch and rethrow here with a regular, untranslatable exception
2) Override the runner to output a "translated" version of the exception error.
3) Change TerminusException to do the replacement itself so that `getMessage()` always returns a valid error message

I favor #3. We'll have to revisit it a bit when we actually start looking at translating error messages, but it should be backwards compatible with the old Terminus runner and forwards compatible with the new one (while not prematurely interpolating error strings).